src/controllers/expenses.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
+
+const expanseServise = require('../services/expenses');
+const userServise = require('../services/users');
+
+const add = (req, res) => {
+  const { userId, spentAt, title, amount, category, note } = req.body;
+
+  const foundUser = userServise.getById(userId);
+
+  const isValidData
+    = foundUser
+    && typeof spentAt === 'string'
+    && typeof title === 'string'
+    && typeof amount === 'number'
+    && typeof category === 'string'
+    && typeof note === 'string';
```

Comment:
the note is not a necessary field, what will be in the case when req.body will came without this field?